examples/CMakeLists.txt

```diff
@@ -20,21 +20,23 @@ function(install_example)
 endfunction()
 
 if(GT_INSTALL_EXAMPLES)
+    message(STATUS "fuck!!! ${GT_INSTALL_EXAMPLES_PATH}")
```

Comment:
Is this a check if I do a proper review?

Reply:
opps. debugging leftover. will fix now